content/applications/finance/accounting/reporting/declarations/tax_returns.rst

```diff
@@ -59,65 +60,55 @@ grids that are used to record invoices or credit notes.
    :alt: Configure taxes and their tax grids in Odoo Accounting
 
 .. note::
-   Taxes and reports are usually already pre-configured: a *Fiscal Localization Package* is
-   installed according to the country you select at the creation of your database. :doc:`Click here
-   <../../fiscal_localizations/overview/fiscal_localization_packages>` for more information about
-   Fiscal Localization Packages.
-
-.. _tax_return_lock:
```

Comment:
why did you delete this anchor?

Reply:
I didn't see the point of it, in this specific point.